AABB_tree/include/CGAL/AABB_tree.h

```diff
@@ -555,8 +577,8 @@ namespace CGAL {
     AABBTraits m_traits;
     // set of input primitives
     Primitives m_primitives;
-    // single root node
-    Node* m_p_root_node;
+    // tree nodes. first node is the root node
+    std::vector<Node> m_p_nodes;
```

Comment:
I suggest to rename `m_p_nodes` to `m_nodes`.

Reply:
The name comes from the original one and there is `m_p_search_tree` (and maybe others) so I'm not in favor of the renaming (if we don't do it for the whole package, which is out of the scope of the PR).

Reply:
That is different: `m_p_nodes` is no longer a pointer. `m_p_search_tree` is still a pointer, and that justifies its prefix `m_p_`.

Reply:
`p_` stands for pointer so you are right.